js/helpers/commonmark.js

```diff
@@ -1,9 +1,9 @@
 import markdownit from 'markdown-it';
 import { options } from 'markdown-it/lib/presets/commonmark';
 
+options.linkify = true;
+const markdown = markdownit(options);
 
 export default function(text) {
```

Comment:
Having to use `.default()` everywhere is a bit ugly, can't we "just" `export mardown.render` or something in this vein?